itk_cucim/filtering/image_grid.py

```diff
@@ -0,0 +1,52 @@
+"""cuCIM accelerated filters for the ITKImageGrid module."""
+import cupy as cp
+import itk
+
+from cucim.skimage.transform import downscale_local_mean
+
+
+def cucim_bin_shrink_image_filter(*args, **kwargs):
+    input_image = args[0]
+    ref_kwargs = kwargs.copy()
+    if 'shrink_factor' in ref_kwargs:
+        ref_kwargs['ShrinkFactor'] = ref_kwargs.pop('shrink_factor')
+    if 'shrink_factors' in ref_kwargs:
+        ref_kwargs['ShrinkFactors'] = ref_kwargs.pop('shrink_factors')
```

Comment:
Unlike on previous PRs, I had to manually swap the case of this here. Is that expected or should there be a change on the ITK side?

`itk.bin_shrink_image_filter` does work with the lowercase `shrink_factors`

Reply:
That's interesting. It should work with both.